File: edudoc/src/app/model/interfaces/custom/claim-voucher.dto.ts
import { IVoucherType } from '../voucher-type';

export interface IClaimVoucherDTO {
    Id: number;
    ClaimEncounterId: number;
    VoucherId: number;
    VoucherAmount: string;
    PaidAmount: string;
    VoucherDate: Date;
    ServiceCode: string,
    ServiceCodeId: number;
    SchoolYear: string;
    SchoolDistrict: string;
    ServiceDate: Date;
    VoucherType: IVoucherType;
    Unmatched: boolean;
    Archived: boolean;
}

File: edudoc/src/Model/DTOs/ClaimVoucherDto.cs
using System.Data.SqlClient;
using System.Data.SqlClient;
using System;

namespace Model.DTOs
{
    public class ClaimVoucherDTO
    {
        public int? ClaimEncounterId { get; set; }
        public int? VoucherId { get; set; }
        public string VoucherAmount { get; set; }
        public string PaidAmount { get; set; }
        public DateTime VoucherDate { get; set; }
        public string ServiceCode { get; set; }
        public int ServiceCodeId { get; set; }
        public string SchoolYear { get; set; }
        public string SchoolDistrict { get; set; }
        public int? SchoolDistrictId { get; set; }
        public DateTime? ServiceDate { get; set; }
        public VoucherType VoucherType { get; set; }
        public bool Unmatched { get; set; }
        public bool Archived { get; set; }
    }
}
